Netflix/Base.lproj/Main.storyboard

```diff
@@ -144,6 +144,12 @@
                                                 <action selector="toggleFullScreen:" target="Ady-hI-5gd" id="dU3-MA-1Rq"/>
                                             </connections>
                                         </menuItem>
+                                        <menuItem title="Enter Picture In Picture" keyEquivalent="p" id="y8W-ni-XGb">
```

Comment:
It would be extra nice if this menu item displayed "Exit Picture in Picture" while actually in PiP mode.

Reply:
Okay!